be/src/vec/exec/format/json/new_json_reader.cpp

```diff
@@ -1818,6 +1816,10 @@ Status NewJsonReader::_simdjson_write_data_to_column(simdjson::ondemand::value&
     } else {
         return Status::InternalError("Not support load to complex column.");
     }
+    // we should set nullmap at last to avoid column_nullable nullmap and data column size not same
```

Comment:
Why it cause problems if it is in the original position?

BTW, use LLM to generate native English comment.

Reply:
According to the original method, we set the nullmap in advance, but if an exception occurs during the logical process of processing data_column below and returns, the size of data_column and nullmap in column_nullable will be inconsistent. Then we catch this exception outter, and call ```_handle_simdjson_error``` which calls column pop_back , in this wrong ColumnNullable , data_column which is nullptr would cause core